xsl/pretext-epub.xsl

```diff
@@ -606,8 +619,8 @@
 <!-- Each must reference an id in the manifest   -->
 <xsl:template name="package-spine">
     <spine xmlns="http://www.idpf.org/2007/opf">
-        <itemref idref="table-contents" linear="yes"/>
         <itemref idref="cover-page" linear="yes" />
+        <itemref idref="table-contents" linear="yes"/>
```

Comment:
This PR as a whole can continue to stew, but this particular edit may be something to enact sooner. Swapping the order of these lines in the spine makes the cover page be the first page when the e-reader is open. As it stands, the cover comes after the TOC.

Reply:
I think this ordering was because at one point, Amazon insisted on having the TOC be the first thing. Apple may have as well. I'm not finding any reference to that, but if time permits while still on break, I can do some testing to see what passes those filters.